nni/runtime/tuner_command_channel/channel.py

```diff
@@ -51,10 +52,19 @@ def disconnect(self) -> None:
 
     def _send(self, command_type: CommandType, data: str) -> None:
         command = command_type.value.decode() + data
-        self._channel.send(command)
+        try:
+            self._channel.send(command)
+        except WebSocket.ConnectionClosed:
+            self._channel = WebSocket(self._url)
+            self._channel.send(command)
 
     def _receive(self) -> tuple[CommandType, str] | tuple[None, None]:
-        command = self._channel.receive()
+        try:
+            command = self._channel.receive()
+        except WebSocket.ConnectionClosed:
+            self._channel = WebSocket(self._url)
```

Comment:
We'd better log this as a warning. Having more logs doesn't hurt.

Do you think retrying immediately and for only once is a good idea? What's the reason for this?